src/EventStore.Core/DataStructures/ILRUCache.cs

```diff
@@ -1,8 +1,13 @@
 using System;
 
 namespace EventStore.Core.DataStructures {
-	public interface ILRUCache<TKey, TValue> {
+	public interface ILRUCache {
+		long Size { get; }
```

Comment:
consider making these explicitly bytes since the same property names in the generic collections are measured in items

Reply:
the `Size` and `Capacity` properties are flexible: they can represent just an item count or also size in bytes (or any other arbitrary value).

For e.g with the default constructor of LRUCache, calculateSize just returns 1 (which represents item count)

Reply:
ah i see, yep fair enough :+1: